readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -44,14 +44,13 @@ function attach_elastic_search_query(data) {
     var project = data.project;
     var version = data.version;
     var language = data.language || 'en';
-    var api_host = data.api_host;
 
     var query_override = function (query) {
         var search_def = $.Deferred();
         var search_url = document.createElement('a');
 
-        search_url.href = api_host;
-        search_url.pathname = '/api/v2/docsearch/';
+        search_url.href = '/';
+        search_url.pathname = data.proxied_api_host + '/api/v2/docsearch/';
```

Comment:
Won't this break with an actual host for proxied_api_host?

Reply:
Yeah, not sure the best way to support both. Maybe checking if it starts with `http` or have two settings (host and path)?

Reply:
I think we can probably just use the approach where we just have the URL instead of splitting the path & search as above: https://github.com/readthedocs/readthedocs.org/pull/6768/files#diff-2caf40ec253a701696aafc199a179c3aL75